packages/common/vbmSignupDeadline.ts

```diff
@@ -42,6 +42,7 @@ const vbmSignupDeadline: StateDeadline = {
   Tennessee: deadline(7, 30),
   Vermont: deadline(8, 4),
   Washington: deadline(7, 28),
+  Virginia: deadline(6, 0),
```

Comment:
I'm assuming this is for primaries because I didn't see anything in October on this list.